blottertrax/database.py

```diff
@@ -7,7 +6,7 @@ class Database:
     sql = None
 
     def __init__(self):
-        self.sql = sqlite3.connect('{}/../database/submissions.db'.format(os.path.dirname(os.path.realpath(__file__))))
+        self.sql = sqlite3.connect('{}/../database/submissions.db')
```

Comment:
Shouldn't need the {} braces if we aren't formatting anything.

Reply:
In fact, this looks like it should maybe just straight be 

../database/submissions

if we remove the format(os...) bit.

Reply:
Getting errors when running this locally